### PR TITLE
tgui: Cyborg Boot Debug

### PR DESCRIPTION
**Pull request by Arkatos1**
## About The Pull Request

This PR refactors Cyborg Boot Debug and brings it's UI from html based one to tgui with some improvements and cleanup. I also took a liberty to add tooltip to each setting to shed some light on its function, as it could be quite unclear what exactly it meant.

**OLD:**

CyborgBootDebugOLD

**NEW:**

CyborgDebugNEW

## Why It's Good For The Game

Better UI.

## Changelog
:cl: Arkatos
refactor: Cyborg Boot Debug now uses tgui. Better check those endoskeletons with a multitool!
/:cl:

### CODE REVIEW

File: tgui/packages/tgui/interfaces/CyborgBootDebug.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+            </LabeledList.Item>
+          </LabeledList>
+        </Section>
+        <Section title="AI Settings">
+          <LabeledList>
+            <LabeledList.Item
+              label="AI Connection Port"
+              buttons={(
+                <Button
+                  icon="info"
```
Comment (MrDoomBringer) [inline tgui/packages/tgui/interfaces/CyborgBootDebug.js:103]:
```suggestion
                  icon="info"
                  color="transparent"
```
This is a bit of personal preference, but I recommend making the info icons transparent. I think it conveys that the button is less for making some action happen and more for being read

**Review by MrDoomBringer — CHANGES_REQUESTED** [review]

File: tgui/packages/tgui/interfaces/CyborgBootDebug.js (hunk 1 above)
Comment (Arkatos1) [reply tgui/packages/tgui/interfaces/CyborgBootDebug.js:103]: Appreciate the advice, but I don't think anyone would think it has some particular function in this particular instance and its easier to get it noticed that way.

**Review by stylemistake — DISMISSED** [review]

**Comment by Cyprex** [comment]
How do you even access this menu?

**Comment by Arkatos1** [comment]
Use a multitool on a finished cyborg endoskeleton. Odd little interaction, but it should be little more known after this PR, as you get a hint now how to access it if you try using endoskeleton in your hand.

**Pull request merged**